samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/evaluation/ExportResponsesBean.java

```diff
@@ -148,6 +148,7 @@ private List<List<Object>> getSpreadsheetData() {
   	  	Iterator detailedStats = histogramListener.getDetailedStatisticsSpreadsheetData(assessmentId).iterator(); 
   	  	detailedStats.next();
   	  	boolean showPartAndTotalScoreSpreadsheetColumns = true;
+  	  	boolean oneSelectionType = totalScores.isOneSelectionType();
```

Comment:
not mandatory change since the JSF will probably need isOneSelectionType as getter, but maybe we can look for a better name.

Something like questionIsOneSelectionType or answerIsOneSelectionType?